docs/blog/rendered/campaign-finance.ipynb

```diff
@@ -0,0 +1,2176 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    comms = conn.create_table("committees", read_committees())
</pre><p>Could use <code>cache()</code> here too if you want. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Oh yeah, this didn't exist when I wrote this.</p>